client/src/components/App/Content/InvestigationInfo/InvestigatedPersonInfo/InvestigatedPersonInfo.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import {Typography, Paper} from '@material-ui/core';
+import {CheckCircleOutline, CakeOutlined, EventOutlined, Help} from '@material-ui/icons';
+import useStyles from './InvestigatedPersonInfoStyles';
+import InfoItemWithIcon from './InfoItemWithIcon';
+
+const InvestigatedPersonInfo = () => {
+    const classes = useStyles();
+    const Divider = () => <span className={classes.divider}> | </span>;
+
+    const name = 'לוי כהן';
+    const investigationId = '2345642132';
+
+    const age = '20';
+    const dateOfTest = '30/01/2020';
+    const gender = 'נקבה';
+    const idType = 'תעודת זהות';
+    const idNumber = '123456789';
+    const isDeceased = 'לא';
+
+    return (
+        <Paper className={classes.paper}>
+            <div className={classes.investigationHeaderInfo}>
+                <CheckCircleOutline color='primary'/>
+                <Typography variant='h6' className={classes.investigationTitle}>
+                    {name}, {investigationId}
+                </Typography>
+            </div>
+
+            <div className={classes.additionalInfo}>
+                <InfoItemWithIcon name='גיל' value={age} icon={CakeOutlined}/>
+                <Divider/>
+                <InfoItemWithIcon name='תאריך הבדיקה' value={dateOfTest} icon={EventOutlined}/>
+                <Divider/>
+                <InfoItemWithIcon name='מין' value={gender} icon={Help}/>
+                <Divider/>
+                <InfoItemWithIcon name='סוג תעודה מזהה' value={idType} icon={Help}/>
+                <Divider/>
+                <InfoItemWithIcon name='מספר תעודה מזהה' value={idNumber} icon={Help}/>
+                <Divider/>
+                <InfoItemWithIcon name='האם נפטר' value={isDeceased} icon={Help}/>
```

Comment:
Use Array of infoItem objects (each includes name, value and icon). 
Map through the array to render InfoItemWithIcon and Divider tags for each object.